src/delegate/printer.rs

```diff
@@ -162,12 +162,17 @@ where
     N: Request,
     N::Params: Serialize,
 {
+    #[derive(Serialize)]
+    struct RawRequest<T: Serialize> {
+        jsonrpc: Version,
+        method: String,
+        params: T,
+        id: Id,
+    }
     // Since these types come from the `lsp-types` crate and validity is enforced via the
     // `Request` trait, the `unwrap()` calls below should never fail.
-    let output = serde_json::to_string(&params).unwrap();
-    let params = serde_json::from_str(&output).unwrap();
-    serde_json::to_string(&request::MethodCall {
-        jsonrpc: Some(Version::V2),
+    serde_json::to_string(&RawRequest {
+        jsonrpc: Version::V2,
         id: Id::Num(id),
         method: N::METHOD.to_owned(),
```

Comment:
By changing the `method` field to `&'static str`, we can eliminate this `to_owned()` call.